### PR TITLE
CORE-8336 save session when user selects to save session from prefere…

### PR DESCRIPTION
**Pull request by sri2k1us**
…nces dialog.

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java [hunk 1]
```diff
@@ -588,9 +588,9 @@ public void saveUserSettings(final UserSettings value,
                                  final boolean updateSilently) {
         final RuntimeCallbacks.SaveUserSettingsCallback callback = new RuntimeCallbacks.SaveUserSettingsCallback(value,
                                                                                userSettings,
-                                                                               announcer,
+                                                                               announcer,this,
                                                                                appearance,
-                                                                               updateSilently);
+                                                                               updateSilently,userSessionService);
```
Comment (psarando) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java]: These new params should be on new lines, so the style is consistent.

Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java]: 👌

Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java]: Requested formatting changes are made!

**Comment by psarando** [comment]
Looks good! :shipit:

File: de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java (hunk 1 above)
Comment (strootman) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java]: @sri2k1us, when making changes like this, within the scope of a PR, it is messy to make new commits for small changes like this.

You could easily merge the commits and force push back to the branch to clean up the history.

Comment (sri2k1us) [inline de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterImpl.java]: Commits merged! Thanks to git.

**Pull request merged**